cfme/tests/configure/test_access_control.py

```diff
@@ -1446,7 +1445,24 @@ def test_tenant_unique_catalog():
         initialEstimate: 1/2h
         startsin: 5.5
     """
-    pass
+    catalog_name = fauxfactory.gen_alphanumeric()
+    catalog_desc = "My Catalog"
+    msg = "Name has already been taken"
+
+    cat1 = appliance.collections.catalogs.create(name=catalog_name, description=catalog_desc)
+    view = navigate_to(appliance.collections.catalogs, 'Add')
+    view.fill({
+        'name': cat1.name,
+        'description': cat1.description
+    })
+    view.add_button.click()
+    view.flash.wait_displayed(timeout=20)
+    assert view.flash.read() == [msg]
+
+    @request.addfinalizer
+    def _delete_catalog():
```

Comment:
Optional:
@mnadeem92 best practice while adding `finalizer`; add it after object creation. It will give readability in bigger test.